java/code/src/com/redhat/rhn/frontend/xmlrpc/org/OrgHandler.java

```diff
@@ -333,26 +333,70 @@ private ChannelFamily lookupChannelFamily(String channelFamilyLabel) {
      *   - One or more of the servers provides do not exist
      *   - The origination or destination organization does not exist
      *   - The user is not defined in the destination organization's trust
+     * @deprecated being replaced by org.transferSystems(User loggedInUser, Integer toOrgId,
```

Comment:
Could you also mention the deprecation in the xmlrpc doc (for all affected methods)?